subt/tf_detector.py

```diff
@@ -29,6 +29,7 @@
 
 class CvDetector:
     def __init__(self):
+        cv2.setNumThreads(2)
```

Comment:
Wunderbar.

I recommend setting a slightly higher value. With two threads, I am getting lots of "dropped" messages in the console on my computer:
`{'name': b'detector', 'delay': 0.114683, 'channel': b'dropped'}`

With three threads, it looks OK, with cpu utilization around 150 %, which is a great improvement over 30000 % before. You may want to go for four threads if you want a safety margin.

Reply:
:-)
I did not see any difference between 3 and 4 threads. It is set to 3 now.